src/main/webapp/onelive/archive/index.html

```diff
@@ -100,9 +100,17 @@ <h1 class="mb-0 text-onelive">Take a look at our past</h1>
             url: 'https://sef-dataholder.herokuapp.com/youtube/onelive',
             dataType: 'json',
             success: function (data) {
+                let newData = data.items;
```

Comment:
It's good if we can use another variable name instead of newData

Reply:
Yeah. Change the variable names or just do not initiate this variable. use `data.items` instead of `newData`

`data.items.ForEach....`

`...html(), {items: data.items}`

Reply:
what about `videoItem`?